src/flag_gems/utils/shape_utils.py

```diff
@@ -246,3 +267,24 @@ def offset_calculator(inp, idx, strides, dim, isInp):
             idx_dim = add_on
         idx = idx // shape[d]
     return offsets if not isInp else (offsets - idx_dim)
+
+
+def offsetCalculator(inp, idx, strides, dim, isInp):
+    ndim = inp.ndim
```

Comment:
are offset_calculator and offsetCalculator the same? you may update the code and reserve one of them.